### PR TITLE
Fix webhook labels

### PR DESCRIPTION
**Pull request by knkski**
Removes label on webhook service that isn't set/used by Juju.

Also extends integration test to deploy mock classifier, waits for it to come up, and then tests that it gets a response.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -49,6 +49,8 @@ jobs:
 
     - name: Run integration tests
       run: tox -vve integration -- --model testing
+      env:
+        KUBECONFIG: /home/runner/.kube/config
```
Comment (ca-scribner) [inline .github/workflows/ci.yaml:53]: Is this necessary because the tests themselves use kubectl/lightkube?

Comment (ca-scribner) [reply .github/workflows/ci.yaml:53]: Also, I think by default kubectl looks in `~/.kube/config`, is that different than this?

File: tests/integration/test_charm.py [hunk 2]
```diff
@@ -18,10 +23,66 @@ async def test_build_and_deploy(ops_test: OpsTest):
...
+    await ops_test.run(
+        "kubectl",
+        "label",
+        "namespace",
+        "default",
+        "serving.kubeflow.org/inferenceservice=enabled",
+        "--overwrite",
+        check=True,
+    )
+
```
Comment (ca-scribner) [inline tests/integration/test_charm.py:35]: Can we change this to use lightkube?  I don't know if there's a convenience function that updates in place like this, but could always get the namespace resource, add your updates to the labels, then submit the resulting labels as a patch

**Comment by ca-scribner** [comment]
Is this a fix for #13?  Or a different issue?

**Comment by ca-scribner** [comment]
Ended up wrapping some suggestions into a PR (#15) onto this branch.  I had some trouble getting things running out of the box locally and I think the suggestion cleans those up, but happy to discuss what should/shouldn't be merged in

**Review by DomFleischmann — APPROVED** [review]
Really glad we are adding these tests.

**Pull request merged**